bundles/org.openhab.binding.ism8/src/main/java/org/openhab/binding/ism8/server/SetDatapointValueMessage.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ism8.server;
+
+import java.nio.ByteBuffer;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link SetDatapointValueMessage} is a message within the KNX frame containing
+ * the information of one data point
+ *
+ * @author Hans-Reiner Hoffmann - Initial contribution
+ */
+@NonNullByDefault
+public class SetDatapointValueMessage {
+    private int id;
+    private byte command;
+    private byte[] data = new byte[0];
+    private byte length;
+
+    public SetDatapointValueMessage() {
+    }
+
+    public SetDatapointValueMessage(byte[] data) throws Exception {
+        if (data.length < 5) {
+            throw new Error("Data size too small for a SetDatapointValueMessage.");
```

Comment:
Throwing `Error` is bad practice, since it signals a fatal error. Better throw a custom exception, where you extend from `Exception`. Same for below.

Reply:
Changed to IllegalArgumentException